src/ui/components/Card/Card.scss

```diff
@@ -35,5 +35,14 @@
   border-bottom-left-radius: $border-radius-default;
   border-bottom-right-radius: $border-radius-default;
   margin-top: 1px;
+  padding: 0;
   text-align: center;
+
+  > a {
```

Comment:
* I think this should apply to all anchors, not just the direct descendant because injecting a footer can be done in so many different ways
* This should also apply to `:link`, `:active`, `:visited`, and `:hover`
* Could you add `font-weight: normal;` while you're in here? The mocks have that

I hit some trouble with the old one too: https://github.com/mozilla/addons-frontend/pull/1586/files#diff-5f781c3e4951964eb41f3c8da5d0cfbdR4